pyomo/contrib/gdpopt/config_options.py

```diff
@@ -35,7 +36,7 @@ def _get_GDPopt_config():
             "need to set subsolver time limits as well."
     ))
     CONFIG.declare("strategy", ConfigValue(
-        default="LOA", domain=In(_supported_strategies),
+        default=None, domain=In(_supported_strategies),
```

Comment:
Why are you taking away the default strategy?

Reply:
Because the strategy that should be used depends heavily on the problem being solved (convex/nonconvex, number of disjunctions, etc). Users should be forced to specify a strategy based on the problem they are solving.